projects/map-ol/src/lib/map-ol.service.ts

```diff
@@ -1001,8 +1142,13 @@ export class MapOlService {
    * for WFS examples.
    */
   private create_wfs_layer(l: VectorLayer): olVectorLayer<olVectorSource<olGeometry>> {
+    let url = null;
```

Comment:
The same code is used in owc-json.service

https://github.com/dlr-eoc/ukis-frontend-libraries/pull/102/commits/67d1f1989cae5b8ffdd949928e6a269dd25422a9